ddtrace/contrib/logging/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""
+The patching of standard library logging to inject tracing information
+
+Example::
+    import logging
+    from ddtrace import tracer
+    from ddtrace import patch_all
+    patch_all(logging=True)
+
+    logging.basicConfig(format='%(asctime)-15s %(message)s - dd.trace_id=%(trace_id)s dd.span_id=%(span_id)s')
```

Comment:
We should standardize what we want to inform people to set here, since they will likely copy/paste the format string.

We want to be sure we have the time, log level, info, filename, line number, message, and trace/span ids. 

We should remove the extra `-` in there too